### PR TITLE
Add search_dashboard job

### PR DESCRIPTION
**Pull request by harterrt**

### CODE REVIEW

File: dags/main_summary.py [hunk 1]
```diff
@@ -129,6 +129,21 @@
...
                        dag=dag)
 
+t12 = EMRSparkOperator(task_id="search_dashboard",
+                       job_name="Search Dashboard",
+                       execution_timeout=timedelta(hours=3),
+                       instance_count=5,
+                       owner="harterrt@mozilla.com",
+                       email=["telemetry-alerts@mozilla.com", "harterrt@mozilla.com"],
+                       env=mozetl_envvar("search_dashboard", {
+                             "start_date": "{{ ds_nodash }}",
```
Comment (acmiyaguchi) [inline dags/main_summary.py]: This should be `submission_date` to reflect the seachdb script, or the script should change the option to `start_date`.

Comment (harterrt) [inline dags/main_summary.py]: Gah, nice catch. Done

**Comment by harterrt** [comment]
Thanks for catching that, Anthony. Silly mistake. RFAL.

I'm going to try and test this locally tomorrow. Docker is giving me fits today.

**Comment by harterrt** [comment]
I was able to test this locally and caught some final syntax errors here: https://github.com/mozilla/telemetry-airflow/pull/154/commits/97fa91b7db43c2a8ca64c6d38a62c940e0b19a1b. 

I'm going to hold off on merging this until I'm back from my PTO next week.

**Pull request merged**